lib/mayaUsd/ufe/UsdUndoConnectionCommands.h

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright 2022 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef MAYAUSD_UFE_USDUNDOCONNECTIONCOMMANDS_H
+#define MAYAUSD_UFE_USDUNDOCONNECTIONCOMMANDS_H
+
+#include <mayaUsd/base/api.h>
+#include <mayaUsd/undo/UsdUndoableItem.h>
+
+#include <ufe/attribute.h>
+#include <ufe/connection.h>
+#include <ufe/path.h>
```

Comment:
Don't think we need this one.

Reply:
Not necessary. I removed it and cleaned up two comments.